### PR TITLE
refactor: :recycle: update the category keywords, to streamline them

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

I was adding categories to the decisions post and saw that there were several that were redundant. So I streamlined them to keep it simpler.

### CODE REVIEW

File: .vscode/json.code-snippets [hunk 1]
```diff
@@ -1,160 +1,137 @@
...
+    "prefix": "decision_categories",
+    "body": [
+      "- backend",
+      "- code snippets",
+      "- communication",
+      "- container",
+      "- contributing",
+      "- collaboration",
+      "- database",
+      "- deploying",
```
Comment (signekb) [inline .vscode/json.code-snippets:118]: Is there some reasoning behind why some categories are in -ing form (is it called the present participle?), like deploying, writing, and licensing, while others are not, like communication and collaboration?

File: .vscode/json.code-snippets [hunk 2]
```diff
@@ -1,160 +1,137 @@
...
+      "${1:Text body}",
+      "",
+      "### ${2:Header}",
+      "",
+      "${3:Text body}",
+      ":::"
+    ],
+    "description": "Insert paneltab (including two tabs here). If you want additional tabs, just include more headers."
+  },
+  // All our key words are included in this code snippet
```
Comment (signekb) [inline .vscode/json.code-snippets:103]:
```suggestion
  // All our category words are included in this code snippet
```

File: .vscode/json.code-snippets (hunk 1 above)
Comment (lwjohnst86) [reply .vscode/json.code-snippets:118]: No reason, but good question! Should we stick to simple present tense verbs? Like deploy, write, collaborate, communicate? Or present continuous like deploying, collaborating?

Comment (signekb) [reply .vscode/json.code-snippets:118]: I think I prefer present tense verbs ("contribute", collaborate", "deploy"), actually. It's shorter than both present continuous ("deploying") and nouns ("deployment").

File: .vscode/json.code-snippets [hunk 3]
```diff
@@ -110,27 +110,27 @@
...
       "- container",
-      "- contributing",
-      "- collaboration",
+      "- contribute",
+      "- collaborate",
       "- database",
-      "- deploying",
+      "- deploy",
       "- dependencies",
       "- development",
```
Comment (signekb) [inline .vscode/json.code-snippets:120]:
```suggestion 
      "- develop",
```
?

File: .vscode/json.code-snippets [hunk 4]
```diff
@@ -110,27 +110,27 @@
...
       "- documentation",
       "- front end",
       "- framework",
       "- git",
       "- installation",
-      "- licensing",
+      "- license",
       "- management",
       "- organization",
       "- standardisation",
```
Comment (signekb) [inline .vscode/json.code-snippets:129]:
```suggestion 
      "- manage",
      "- organize",
      "- standardise",
``` 
?? Or am I going too far?

**Review by signekb — COMMENTED** [review]
Nice 👍 I have added some suggestions, but not sure whether I'm taking it too far. What do you think?

File: .vscode/json.code-snippets (hunk 4 above)
Comment (lwjohnst86) [reply .vscode/json.code-snippets:129]: hahah I was actually thinking the same thing. Maybe? Would make it all consistent.

File: .vscode/json.code-snippets [hunk 5]
```diff
@@ -1,160 +1,137 @@
...
+      "- code snippets",
+      "- communicate",
+      "- container",
+      "- contribute",
+      "- collaborate",
+      "- database",
+      "- deploy",
+      "- dependencies",
+      "- develop",
+      "- documentation",
```
Comment (lwjohnst86) [inline .vscode/json.code-snippets:121]:
```suggestion
      "- document",
```

File: .vscode/json.code-snippets [hunk 6]
```diff
@@ -1,160 +1,137 @@
...
+      "- front end",
+      "- framework",
+      "- git",
+      "- installation",
```
Comment (lwjohnst86) [inline .vscode/json.code-snippets:125]:
```suggestion
      "- install",
```

**Pull request merged**